lifemonitor/api.yaml

```diff
@@ -137,36 +351,70 @@ components:
       schema:
         type: string
       required: true
-    test_suite:
-      name: "test_suite"
-      description: "Flag to retrieve test suites info of the workflow"
-      in: query
-      schema:
-        type: boolean
-      required: false
-    test_build:
-      name: "test_build"
-      description: "Flag to retrieve test build info of test suites"
-      in: query
-      schema:
-        type: boolean
-      required: false
-    test_output:
-      name: "test_output"
-      description: "Flag to retrieve output of workflow tests"
-      in: query
-      schema:
-        type: boolean
-      required: false
     suite_uuid:
       name: "suite_uuid"
       description: "TestSuite's Unique identifier"
       in: path
       schema:
         type: string
       required: true
+    instance_id:
+      name: "instance_id"
+      description: "Test service Instance Unique identifier"
+      in: path
+      schema:
+        type: string
+      required: true
+    build_id:
+      name: "build_id"
+      description: "Build identifier"
+      in: path
+      schema:
+        type: string
+      required: true
+    offset:
```

Comment:
Not sure I understand this correctly. Wouldn't users want to get the N latest builds rather than skip the first N? Also, to specify an offset and a limit they'd need to know the total number of builds in advance, wouldn't they?

Reply:
The idea here was to allow pagination.  This feature may not be necessary, so we could opt to only support retrieving the last `n` up to a maximum.